src/input/NewButton.tsx

```diff
@@ -0,0 +1,77 @@
+import { Button as MaterialUiButton, Grid, makeStyles, Avatar, Typography } from '@material-ui/core';
+import React from 'react';
+
+
+interface TileButtonProps {
+    title?: string
+    subTitle?: string
+    percent?: string
+    backgroundColor?: string
+    color?: string
+    selected?: boolean
+}
+
+const useStyles = makeStyles((theme) => ({
+    buttonSelected: {
+        backgroundColor: 'white',
+        color: '#38b2cc',
```

Comment:
I found out that the brandbook colors are slightly outdated. You can use the theme here directly. 

color: theme.palette.secondary.main,